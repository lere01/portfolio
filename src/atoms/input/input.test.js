import React from "react";
import renderer from "react-test-renderer";
import MInput from "./inputs";
import MTextarea from "./textarea";

describe("describe the different inputs", () => {
  it("Renders correctly when called", () => {
    const tree = renderer.create(<MInput />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it("Renders correctly when called", () => {
    const tree = renderer.create(<MTextarea />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
