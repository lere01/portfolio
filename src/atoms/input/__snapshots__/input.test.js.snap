// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly when called 1`] = `
<input
  class="my-input"
  type="text"
/>
`;

exports[`describe the different inputs Renders correctly when called 1`] = `
<input
  class="my-input"
  type="text"
/>
`;

exports[`describe the different inputs Renders correctly when called 2`] = `
<textarea
  class="my-input"
  cols="4"
  rows="5"
/>
`;

exports[`hello Renders correctly when called 1`] = `
<input
  class="my-input"
  type="text"
/>
`;
